### ISSUE TITLE
Implementation for RP2040

### ISSUE THREAD

**Issue by SimonGAndrews** [nodejs/help#3272]
Is there any chance of an implementation  of node.js on the RP2040  microcontroller.  ????
If so could we have a placeholder for an enhancement ?..

Regards Simon 
https://en.m.wikipedia.org/wiki/RP2040

**Comment by addaleax** [comment]
https://github.com/neonious/lowjs is not 1:1 Node.js, but it might be giving you what you need?

**Comment by gireeshpunathil** [comment]
closing as answered, pls reopen if there is anything outstanding.

**Comment by SimonGAndrews** [comment]
Thank you for considering. I investigated lowjs in detail and it is an option for node like framework for the esp32 microcontroller family.   An issue is it is not fully open source and is highly dependent upon authors distribution infrastructure, but it will provide A good JavaScript interpreter and ran a number of NPM libraries which I tried. 
My preferred option / conclusion was to use the Espruino JS Interpreter `https://www.espruino.com/ `  .  It does not run all modern JS functions (for good reason .. to not bloat micro resources ) and is not a full node substitute or run any NPM packages,. But it is fully open source, runs on many microcontrollers, accommodates modules structure. It's a very good JS Interpreter with a supportive community. Thanks again.
